packager/live_packager_test.cc

```diff
@@ -942,4 +942,58 @@ INSTANTIATE_TEST_CASE_P(
             5, "audio/en/init.mp4", LiveConfig::EncryptionScheme::SAMPLE_AES,
             LiveConfig::OutputFormat::TS, LiveConfig::TrackType::AUDIO,
             "audio/en/%05d.m4s", false}));
+
+struct TimedTextTestCase {
+  const char* media_segment_format;
+  LiveConfig::OutputFormat output_format;
+};
+
+class TimedTextParameterizedTest
+    : public LivePackagerBaseTest,
+      public ::testing::WithParamInterface<TimedTextTestCase> {
+  void SetUp() override {
+    LivePackagerBaseTest::SetUp();
+
+    LiveConfig live_config;
+    live_config.format = GetParam().output_format;
+    live_config.track_type = LiveConfig::TrackType::TEXT;
+    live_config.protection_scheme = LiveConfig::EncryptionScheme::NONE;
+    SetupLivePackagerConfig(live_config);
+  }
+};
+
+TEST_P(TimedTextParameterizedTest, VerifyTimedText) {
```

Comment:
It would be great if we could also add a negative test, for example, what if client specifies `track_type != LiveConfig::TrackType::TEXT`?